client/src/app/pages/license-list/license-table.tsx

```diff
@@ -0,0 +1,163 @@
+    currentPageItems,
+    propHelpers: {
+      paginationProps,
+      tableProps,
+      getThProps,
+      getTrProps,
+      getTdProps,
+    },
+  } = tableControls;
+
+  return (
+    <>
+      <Table {...tableProps} aria-label="License table">
+        <Thead>
+          <Tr>
+            <TableHeaderContentWithControls {...tableControls}>
+              <Th {...getThProps({ columnKey: "name" })} />
+              <Th {...getThProps({ columnKey: "packages" })} />
+              <Th {...getThProps({ columnKey: "sboms" })} />
+            </TableHeaderContentWithControls>
+          </Tr>
+        </Thead>
+        <ConditionalTableBody
+          isLoading={isFetching}
+          isError={!!fetchError}
+          isNoData={totalItemCount === 0}
+          numRenderedColumns={numRenderedColumns}
+        >
+          {currentPageItems.map((item, rowIndex) => {
+            return (
+              <Tbody key={item.license}>
+                <Tr {...getTrProps({ item })}>
+                  <TableRowContentWithControls
+                    {...tableControls}
+                    item={item}
+                    rowIndex={rowIndex}
+                  >
+                    <Td
+                      width={35}
+                      modifier="breakWord"
+                      {...getTdProps({
+                        columnKey: "name",
+                        item: item,
+                        rowIndex,
+                      })}
+                    >
+                      {item.license}
+                    </Td>
+                    <Td
+                      width={15}
+                      {...getTdProps({
+                        columnKey: "packages",
+                        item: item,
+                        rowIndex,
+                      })}
+                    >
+                      <WithPackagesByLicense
+                        key={item.license}
+                        licenseId={item.license}
+                      >
+                        {(totalPackages, isFetching, fetchError) => (
+                          <LoadingWrapper
+                            isFetching={isFetching}
+                            fetchError={fetchError}
+                            isFetchingState={
+                              <Skeleton screenreaderText="Loading contents" />
+                            }
+                            fetchErrorState={(error) => (
+                              <TableCellError error={error} />
+                            )}
+                          >
+                            {totalPackages && totalPackages > 0 ? (
+                              <NavLink
+                                to={generatePath(Paths.packagesFilter, {
+                                  licenseName: item.license,
+                                })}
+                              >
+                                {`${totalPackages} Package${totalPackages > 1 ? "s" : ""}`}
+                              </NavLink>
```

Comment:
Let's remove the `<Navlink/>` please. We know we need it but for this PR It is better not to include it, then a new PR could add the <Navlink/>